man/resolved.conf.xml

```diff
@@ -225,6 +225,12 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>mDNS=</varname></term>
+        <listitem><para>Takes a boolean argument. If <literal>yes</literal> (the default), mDNS stack is enabled.
+        If <literal>no</literal>, the mDNS stack is disabled.</para></listitem>
```

Comment:
Even if per-network MulticastDNS is off, resolved open port 5353 and conflict with other mDNS responder e.g. avahi-daemon. If `mdns_support` of the resolved manager is off, then resolved does not try to open port 5353, and avahi-daemon can safely start. However, currently `mdns_support` variable of the resolved manager is always on. So, in this commit I add the option to control `mdns_support` variable.